docker/dev-ticker/Dockerfile

```diff
@@ -1,19 +1,20 @@
 # syntax=docker/dockerfile:experimental
-FROM rust:1.59 as builder
+FROM rust:1.68 as builder
 RUN --mount=type=cache,target=/usr/local/cargo/registry \
-    cargo install sccache
-WORKDIR /usr/src/zksync
+    cargo install sccache --no-default-features
+WORKDIR /usr/src/rollup
 COPY . .
 RUN --mount=type=cache,target=/usr/local/cargo/registry \
     --mount=type=cache,target=/root/.cache/sccache \
     RUSTC_WRAPPER=/usr/local/cargo/bin/sccache \
-    cargo build --bin dev-ticker-server --release
+    cargo build --bin dev_ticker_server --release
 
 FROM debian:bullseye
 RUN apt update
 RUN apt install openssl -y
 EXPOSE 9876
+ENV RUST_BACKTRACE "full"
 ENV RUST_LOG info
-COPY --from=builder /usr/src/zksync/target/release/dev-ticker-server /bin/
-COPY --from=builder /usr/src/zksync/etc/tokens /etc/tokens
-ENTRYPOINT ["dev-ticker-server"]
+COPY --from=builder /usr/src/rollup/target/release/dev_ticker_server /bin/
+COPY --from=builder /usr/src/rollup/etc/tokens /etc/tokens
```

Comment:
Do we want to copy this, or create a volume? Or both?

Reply:
I'm not sure what you mean. here we just copy it from the builder.

Reply:
maybe we can do it in a different task, improving the Dockerfile. This copy the tokens file but its better to have a volume with it.

Reply:
In this specific case, I don't see the benefit of using a volume.

Reply:
By not using a volume, when I run the container without the volume, its going to use the localhost, if it exists, when the image was created. This container its just for local deployment, isn't it?

Reply:
so I removed the copy

Reply:
we can discuss it, after the daily, I will leave it there.